components/checkbox/stories/checkbox.test.js

```diff
@@ -29,6 +31,11 @@ export const CheckboxGroup = Variants({
 			testHeading: "Invalid",
 			isInvalid: true,
 		},
+		{
+			testHeading: "Invalid, checked",
+			isInvalid: true,
+			isChecked: true,
+		},
```

Comment:
I added this test case, but wasn't completely sure if I need to go down this road. Do we need Chromatic coverage for variants, like invalid-checked, readOnly-checked, disabled-checked, etc?

Reply:
I'm on the fence for this one. The checkbox background color _does_ change across the invalid/disabled etc. variants, but you don't actually need the box to be checked to see the background color change, but on the other hand, it does make it a lot more obvious if checked and unchecked are both failing a check because of their background color.

Reply:
If there's a possibility for a regression in these 2 settings existing together, I definitely support adding the extra test case.